From cc9dd9c99d79767e66f8fc8e30d2e9dcb54ce559 Mon Sep 17 00:00:00 2001
From: drosdeck <edson.drosdeck@gmail.com>
Date: Mon, 9 Jan 2023 11:56:00 -0400
Subject: [PATCH] Fix Positivo MASTER-N1110 key toggle touchpad

(cherry picked from commit 6152de293ae2c363ecc1441129fd0f916cbe40fe)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 4b58a42232..169e3e6f66 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2026,6 +2026,8 @@ evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:
 ###########################################################
 # Positivo
 ###########################################################
+# Positivo MASTER-N1110 
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:svnPositivoTecnologiaSA:pn*:pvr*:rvnPositivoTecnologiaSA:rnNP11G-E*
 # Positivo DUO (k116)
 evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:svnPositivoTecnologiaSA:pn*:pvr*:rvnPositivoTecnologiaSA:rnK116*
  KEYBOARD_KEY_76=f21                                    # Fn+F1 toggle touchpad
